app/api/discount_codes.py

```diff
@@ -98,12 +130,19 @@ def query(self, view_kwargs):
             else:
                 raise ForbiddenException({'source': ''}, 'Event organizer access required')
 
+        # discount_code - ticket :: many-to-many relationship
+        if view_kwargs.get('ticket_id') and has_access('is_coorganizer'):
+            self.schema = DiscountCodeSchemaTicket
+            ticket = safe_query(
+                self, Ticket, 'id', view_kwargs['ticket_id'], 'ticket_id')
```

Comment:
`ticket = safe_query()`. Make it on the same line. That's the convention we're using else where in the project. Please don't mix the conventions.